packages/itinerary-body/src/defaults/access-leg-step.tsx

```diff
@@ -104,10 +105,13 @@ export default function AccessLegStep({
     // for styled-components support.
     <span className={className} style={style}>
       {stepContent}
-      {/* TODO: Implement metric vs imperial (up until now it's just imperial). */}
       {step?.distance > 0 && (
         <S.StepLength>
-          {humanizeDistanceString(step.distance, false, intl)}
+          {humanizeDistanceString(
+            step.distance,
+            coreUtils.metricUnits.areMetricUnitsUsed(intl.locale),
+            intl
```

Comment:
Would it be possible to refactor `humanizeDistanceString` to use the `intl` object it gets anyway to avoid having to import `coreUtils` everywhere?

Reply:
I'd say don't remove the manual override (for backwards compatibility) but instead use it as an override -- a way to force metric even if the locale isn't metric